src/deluge/playback/mode/session.cpp

```diff
@@ -1515,7 +1522,8 @@ void Session::armClipsToStartOrSoloWithQuantization(uint32_t pos, uint32_t quant
 
 	// If we were doing this just for one Clip (so a late-start might be allowed too)...
 	if (clip) {
-		if (clip->launchStyle == LaunchStyle::DEFAULT) {
+		if (clip->launchStyle == LaunchStyle::DEFAULT || (clip->launchStyle == LaunchStyle::FILL && !doLateStart)
```

Comment:
Why aren't once and fill allowed to do late starts? Also this expression can just be DEFAULT||!doLateStart

Reply:
The late starts don't trigger launch events for later. This was a bug from previous Fill mode, if you do Shift + audition, they will start playing and never be stopped after that.

Reply:
simplifying still applies

Reply:
Ok I'll modify it